plugins/inventory/opennebula.py

```diff
@@ -143,7 +143,8 @@ def _get_vm_ipv4(self, vm):
             nic = [nic]
 
         for net in nic:
-            return net['IP']
+            ip = net.get('IP')
+            return ip
```

Comment:
If you want to match IPv6 behavior, it should be:
```suggestion
            if ip:
                return ip
```